package.json

```diff
@@ -66,31 +65,29 @@
     ]
   },
   "resolutions": {
-    "@types/react": "18.0.15",
     "node-fetch": "^2.6.7",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "ts-node": "10.9.1"
   },
   "devDependencies": {
-    "check-dependency-version-consistency": "3.0.0",
-    "concurrently": "7.0.0",
+    "check-dependency-version-consistency": "3.0.3",
+    "concurrently": "7.5.0",
     "cross-env": "7.0.3",
     "dotenv-flow": "3.2.0",
-    "husky": "7.0.4",
-    "lint-staged": "12.3.3",
-    "lockfile-lint": "4.6.2",
+    "husky": "8.0.2",
+    "lint-staged": "13.0.3",
+    "lockfile-lint": "4.9.6",
     "markdownlint-cli": "0.32.2",
     "npm-run-all": "4.1.5",
-    "patch-package": "6.4.7",
+    "patch-package": "6.5.0",
     "postinstall-postinstall": "2.1.0",
-    "prettier": "2.5.1",
-    "prettier-plugin-packagejson": "2.2.15",
-    "prettier-plugin-sh": "0.8.1",
-    "serve": "13.0.2",
```

Comment:
Introduced in https://github.com/hashintel/hash/commit/a2e2e6fa402b03e1e58646603f0d3662c22f4707. No longer needed because we use `block-scripts` / local `serve` in blocks.